src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationOptions.cs

```diff
@@ -29,5 +29,52 @@ public AlipayAuthenticationOptions()
         ClaimActions.MapJsonKey(Claims.Gender, "gender");
         ClaimActions.MapJsonKey(Claims.Nickname, "nick_name");
         ClaimActions.MapJsonKey(Claims.Province, "province");
+        ClaimActions.MapJsonKey(Claims.OpenId, "open_id");
+        ClaimActions.MapJsonKey(Claims.UserId, "user_id");
+    }
+
+    /// <summary>
+    /// Get or set a value indicating whether to use certificate mode for signature implementation.
+    /// <para>https://opendocs.alipay.com/common/057k53?pathHash=e18d6f77#%E8%AF%81%E4%B9%A6%E6%A8%A1%E5%BC%8F</para>
+    /// </summary>
+    public bool EnableCertSignature { get; set; }
+
+    /// <summary>
+    /// Gets or sets the optional ID for your Sign in with app_cert_sn.
+    /// </summary>
+    public string? AppCertSNKeyId { get; set; }
```

Comment:
Assuming that's what `SN` means here:

```suggestion
    public string? ApplicationCertificateSubjectNameKeyId { get; set; }
```